specification/langRef/base/mapresources.dita

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE reference PUBLIC "-//OASIS//DTD DITA Reference//EN" "reference.dtd">
+    <reference id="mapresources" xml:lang="en-us">
+      <title><xmlelement>mapresources</xmlelement></title>
+      <shortdesc>The <xmlelement>mapresources</xmlelement> element serves as an intuitive location
+        for map authors to place objects with <xmlatt>processing-role</xmlatt> set to
+          <keyword>resource-only</keyword>, for example, key definitions and subject scheme
+        maps</shortdesc>
+      <prolog>
+        <metadata>
+          <keywords>
+            <indexterm>convenience
+                elements<indexterm><xmlelement>mapresources</xmlelement></indexterm></indexterm>
+            <indexterm>mapgroup
+              domain<indexterm><xmlelement>mapresources</xmlelement></indexterm></indexterm>
+          </keywords>
+        </metadata>
+      </prolog>
+      <refbody>
+        <section id="specialization-hierarchy">
+          <title>Specialization hierarchy</title>
+          <p>The <xmlelement>mapresources</xmlelement> element is specialized from
+              <xmlelement>topicref</xmlelement>. It is defined in the map group module.</p>
+        </section>
+        <section id="attributes">
+          <title>Attributes</title>
+          <p>The following attributes are available on this element: <xref
+                    keyref="attributes-universal"/>, <xref keyref="attributes-linkRelationship"/>
+                (with the exclusion of <xmlatt>chunk</xmlatt>, <xmlatt>collection-type</xmlatt>,
```

Comment:
I think the "exclusion" here is on the wrong group, chunk / collection-type / locktitle are not part of the linkRelationship group